src/test/ui/lint/rfc-2457-non-ascii-idents/lint-mixed-script-confusables.rs

```diff
@@ -0,0 +1,9 @@
+#![feature(non_ascii_idents)]
+#![deny(mixed_script_confusables)]
+
+struct ΑctuallyNotLatin;
+//~^ ERROR The usage of Script Group `AugmentedScriptSet {Grek}` in this crate consists solely of
```

Comment:
Debug impls should not be used here, we should instead have something that will display `Greek` and `Greek + Foo + Bar` without the `AugmentedScriptSet {}` bit. `ScriptExtension` already has a Debug impl that could be used for this. I can instead give it a Display impl that's better here.

Reply:
https://github.com/unicode-rs/unicode-security/pull/21